### ISSUE TITLE
Open tabs and history not synchronized. Open tabs in workspaces should also be synchronized.

### ISSUE THREAD

**Issue by universish** [zen-browser/desktop#3498]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

 Open tabs and history cannot be synchronized between zen browsers on two PCs.
History cannot be synchronized. Mozilla Firefox's history synchronization and open tabs synchronization does not work.
Also open tabs in Workspaces should be synchronized within the same workspace. 
Also workspaces should be synchronized. Workspaces in both browsers should be synchronized and workspaces names and icons should be synchronized. The workspaces synchronization feature is available in the firefox synchronization system. But I am not sure if it works or not.

Translated with DeepL.com (free version)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.0 (64-bit)

### What platform are you seeing the problem on?

Linux, Windows

Both devices are Windows 11. Another device is fedora 41. zen-browser is installed on the nixOS virtual machine. I need to sync all of these. But it doesn't work.

**Comment by rihards-simanovics** [comment]
Just Installed Zen browser on intel iMac and realised that although you can select to sync workspaces the tabs within them are not synchronised. Is there a chance we could get this in the future?

My main workstation is running Windows 11 Enterprise. The iMac is an older intel late 2017 21.5" model so Ventura 13.7.1 is all it can handle now.

**Comment by cmdillon** [comment]
I have the same issue. the tabs in workspaces on different devices keeps getting zeroed—meaning if I have tabs open in one device and sync, then open zen on another device the workspace will be empty, in some cases it will remove the tabs.

**Comment by tptodorov** [comment]
Same on Mac os x, intel version.

**Comment by ManuelHalbwachs** [comment]
Same here, 1.7.6.b

**Comment by SamCao920** [comment]
Same issue. The tabs do not sync between devices, only workspaces do.

**Comment by slw1jw** [comment]
I have the same issue trying to sync open tabs. Workspaces sync fine. Zen Mods don't sync neither, which they should.

**Comment by palinko-d** [comment]
It would be awesome to have all active tabs mirrored to all windows within the same workspace - either on one machine or across all devices (this is how the Arc browser works).
Firefox sync is functioning - you can see tabs in "Show tabs from other devices" menu, but it doesn't show workspaces at all and it's not very pleasant to work with that.

**Comment by rihards-simanovics** [comment]
> It would be awesome to have all active tabs mirrored to all windows within the same workspace - either on one machine or across all devices (this is how the Arc browser works). Firefox sync is functioning - you can see tabs in "Show tabs from other devices" menu, but it doesn't show workspaces at all and it's not very pleasant to work with that.

Yeah, this would be awesome. It kind of struck me when I noticed that if you open two windows with the same workspace, the tabs are not duplicated. Also, having "Ad-hock" windows would also be good. There are times when I need to have a window with just one tab as I'm doing some copying and pasting or other work where split tabs won't work.

**Comment by bdonohoe** [comment]
This is the bug that keeps me from switching to Zen Browser. I don't get any tab syncing at all, just a bunch of empty workspaces. Syncing the existence of workspaces without their contents is useless. I hope this gets some attention! That it has been sitting untriaged for four months is worrisome.

**Comment by marcorossiIT** [comment]
> I have the same issue. the tabs in workspaces on different devices keeps getting zeroed—meaning if I have tabs open in one device and sync, then open zen on another device the workspace will be empty, in some cases it will remove the tabs.

Could this be an Enhancement type of issue? I see no other browser is expected to behave like this right? I'd like this one anyway. I'd also like to have all the pages opened on multiple windows btw but now i got offtopic.

**Comment by HarunKilic** [comment]
> > I have the same issue. the tabs in workspaces on different devices keeps getting zeroed—meaning if I have tabs open in one device and sync, then open zen on another device the workspace will be empty, in some cases it will remove the tabs.
> 
> Could this be an Enhancement type of issue? I see no other browser is expected to behave like this right? I'd like this one anyway. I'd also like to have all the pages opened on multiple windows btw but now i got offtopic.

Microsoft Edge and Arc provides this.

**Comment by mr-cheffy** [comment]
## Closing this issue as part of a repo cleanup

Due to technical-debt we had in the past, we are currently clearing out old, duplicate, stale, and non-actionable issues to improve issue tracking and make it easier to focus on fixing real bugs.

If this issue is still relevant, please open a new issue using our updated issue template, which will help us categorize and address it more efficiently.

Thanks for your understanding and for helping us improve the project! 🚀

This issue can be reopened at: https://github.com/zen-browser/desktop/issues/new?template=bug_report.yml&title=Open%20tabs%20and%20history%20not%20synchronized.%20Open%20tabs%20in%20workspaces%20should%20also%20be%20synchronized.%0A

**Comment by bdonohoe** [comment]
Seriously? This is receiving new comments as recently as yesterday and it gets swept up in an "old, duplicate, stale, and non-actionable" purge? Way to disrespect all the people who already weighed in on this issue.

**Comment by HarunKilic** [comment]
> Seriously? This is receiving new comments as recently as yesterday and it gets swept up in an "old, duplicate, stale, and non-actionable" purge? Way to disrespect all the people who already weighed in on this issue.

Indeed. Pretty weird closing an active issue without valid reason.

**Comment by georg-stone** [comment]
> > Seriously? This is receiving new comments as recently as yesterday and it gets swept up in an "old, duplicate, stale, and non-actionable" purge? Way to disrespect all the people who already weighed in on this issue.
> 
> Indeed. Pretty weird closing an active issue without valid reason.

Do you realize that the Zen lead maintainer is in _college_? I don't think you'd want to maintain a BROWSER with one thousand issues. His purge is completely justified since he is trying not to be so overwhelmed. As he said, if the problem is still present, open it again.

**Comment by rihards-simanovics** [comment]
@bdonohoe & @HarunKilic, please don't worry. As the comment says, the issue is not abandoned. They mass closed the issues so they could start from a clean slate, addressing only new and existing issues, which is why they encouraged the creation of new issues.

> If this issue is still relevant, please open a new issue using our updated issue template, which will help us categorize and address it more efficiently.

As someone who deals with Issues in open-source projects, I am a maintainer for InvocieShelf, and I can vouch that this is just to save themselves from headaches, as going through 1000+ issues is unfeasible.

I've reopened this issue here, #6981, with a reference to this issue, so nothing is lost.

**Comment by 0Ky** [comment]
@georg-stone It feels like the maintainer is just trying to clear out the backlog without actually addressing the real problems, which kinda brushes off everyone who's put in the effort to report and discuss these issues. I'm sure he's only trying to keep things manageable...

The issue with closing active issues without real context or valid reason is that it disrespects the people who've already contributed to the conversation as @bdonohoe mentioned. Creating a new issue erases the entire history of comments and discussions. It makes it harder to track ongoing problems and build on what's already been said. 

Issues were already being labelled as `Stale` automatically and _closed_ when no further comments were made by the **dosubot**. The point is, if it's really about improving issue tracking, there should have been a more thoughtful approach to just nuking all of the issues. Other alternatives would include having the bot close duplicates, close and mark certain issues with a `Retest` label and then reopen them once they've been retested and verified with the latest version would have been much better instead of asking people to re-create a new issue of the exact same issue.

**Comment by georg-stone** [comment]
So, if you had a BROWSER downloaded by millions of people with 1k issues, what would be your solution? Just reopen the issue and include the ACTUALLY important context from the comments.

**Comment by rihards-simanovics** [comment]
Ok, guys, let's take a break from the comments. There will always be people like me who will reopen the issue if it remains relevant. I've seen these kinds of issues reset on big projects before; it always generates some backlash, but things do die down later. So, please, let's calm things down and stay clear-minded. I've reopened the issue, so there is no problem.

**Comment by starfishpatkhoo** [comment]
I think the gist of this issue has moved to https://github.com/zen-browser/desktop/issues/6981 and then from there to https://github.com/zen-browser/desktop/discussions/7512 ...

**Comment by mr-cheffy** [comment]
The comment explains it, lots of duplicates, issues made by mods/CSS and alpha-stage related without any sort of organisation. A clear up when entering a new development cycle was the smartest choice to enhance productivity. Thanks
